### DISCUSSION TITLE
Question regarding custom pickup time for paper

### DISCUSSION THREAD

**Discussion by kevincrosser** [xirixiz/homeassistant-afvalwijzer#270] [General]
Hi,

I have a question regarding paper pickup. This is not provided by the api. The Pickup time is every first wednessday of the month. Is there a way to provide this data to your add-on?

**Comment by PatrickSt1991** [comment]
@kevincrosser is it still not provided by the API? what's your garbage collector? maybe I can figure something out for you

**Comment by sebastiaanspeck** [comment]
If this is still an issue, please open an issue instead of a discussion.
